typespec-extension/src/main/java/com/azure/autorest/fluent/TypeSpecFluentNamer.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.autorest.fluent;
+
+import com.azure.autorest.TypeSpecPlugin;
+import com.azure.autorest.fluentnamer.FluentNamer;
+
+import java.lang.reflect.Type;
+
+public class TypeSpecFluentNamer extends FluentNamer {
+    public TypeSpecFluentNamer() {
+        super(new TypeSpecPlugin.MockConnection(), "dummy", "dummy");
+    }
+
+    @Override
+    public <T> T getValue(Type type, String key) {
+        // Logger uses getValue to get log level, null will use default
+        return null;
+    }

```

Comment:
We had some JavaSettings check in FluentNamer e.g. `JavaSettings.getInstance().isGenerateClientInterfaces()`

Will it safer if we have the same MAP here from `TypeSpecFluentPlugin`?

Reply:
OK, there's FluentJavaSettings instance in `transform` method. We should use the same map here.